plugin/ochttp/server_test.go

```diff
@@ -595,3 +595,63 @@ func TestIgnoreHealthz(t *testing.T) {
 		t.Errorf("Got %v spans; want no spans", spans)
 	}
 }
+
+func testHealthEndpointSkipArray(p string) bool {
+	for _, toSkip := range []string{"/health", "/metrics"} {
+		if p == toSkip {
+			return true
+		}
+	}
+	return false
+}
+
+func TestHandlerIsHealthEndpoint(t *testing.T) {
+	var spans int
+
+	client := &http.Client{}
+	tests := []struct {
+		path               string
+		healthEndpointFunc func(string) bool
+	}{
+		{"/healthz", nil},
+		{"/_ah/health", nil},
+		{"/health", testHealthEndpointSkipArray},
+		{"/metrics", testHealthEndpointSkipArray},
```

Comment:
Need a test case for 
```
		{"/healthz", testHealthEndpointSkipArray},
		{"/_ah/health", testHealthEndpointSkipArray},
```

Also, TestIgnoreHealthz is not required as it is covered under this test. May be make this a generic name as it covers all cases.